src/App.css

```diff
@@ -1,3 +1,310 @@
 .App {
   font-family: Arial, Helvetica, sans-serif;
 }
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 10px;
+  background-color: #fff;
+}
+
+.projectHeading {
+  width: auto;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-size: 1.175rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.menuitem {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 1;
+  text-decoration: none;
+}
+
+.bookContainer {
+  background-color: #f5f6fa;
+  margin: 10px;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  height: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.mainBookSection {
+  display: flex;
+  flex-direction: column;
+}
+
+.categoryTitle {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.bookTitle {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.authorName {
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 1.3075rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin-top: 10px;
+  margin-bottom: 20px;
+}
+
+.bookButton {
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 0.975rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  margin-top: 10px;
+  color: #4386bf;
+  padding: 5px;
+  padding-right: 10px;
+  padding-left: 10px;
+  cursor: pointer;
+  border: none;
+}
+
+.specialButton {
+  border-right: 1px solid black;
+  background-color: #f5f6fa;
+}
+
+.buttons {
+  margin-top: 10px;
+}
+
+.bookButton:hover {
+  color: #1f4464;
+}
+
+.Form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #b5b8c5;
+}
```

Comment:
- [ ] As requested previously , you need to ensure that the design of the form ( The arrangement of fields, "Add Book" Button and background color ) should match the Zeplin design. Cheers 👍  

Zeplin Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/202865668-b7d71242-d81c-4dcf-932b-6bdc1c41541e.png)  |  ![image](https://user-images.githubusercontent.com/60618877/202865688-284fc78d-7c4a-4127-97e9-77f36981e811.png)